exts/cesium.omniverse/cesium/omniverse/usdUtils/usdUtils.py

```diff
@@ -42,75 +74,84 @@ def add_tileset_ion(name: str, tileset_id: int, token: str = "") -> str:
 def add_imagery_ion(tileset_path: str, name: str, asset_id: int, token: str = "") -> str:
     stage = omni.usd.get_context().get_stage()
 
-    safe_name = re.sub("[\\W]+", "_", name)
+    safe_name = get_safe_name(name)
 
-    imagery_path: str = Sdf.Path(tileset_path).AppendPath(safe_name)
+    imagery_path = Sdf.Path(tileset_path).AppendPath(safe_name).pathString
 
-    # get_stage_next_free_path will increment the path name if there is a colllision
-    imagery_path: str = Sdf.Path(omni.usd.get_stage_next_free_path(stage, imagery_path, False))
+    # get_stage_next_free_path will increment the path name if there is a collision
+    imagery_path = omni.usd.get_stage_next_free_path(stage, imagery_path, False)
 
     imagery = CesiumIonImagery.Define(stage, imagery_path)
-    assert imagery.GetPrim().IsValid()
-    parent = imagery.GetPrim().GetParent()
-    assert parent.IsA(CesiumTileset)
 
     imagery.GetIonAssetIdAttr().Set(asset_id)
     imagery.GetIonAccessTokenAttr().Set(token)
 
     server_prim_path = get_path_to_current_ion_server()
-    if server_prim_path != Sdf.Path.emptyPath:
+    if server_prim_path != "":
```

Comment:
In general, we're treating prim paths as strings. What's the motivation for using strings over the `Sdf.Path` class?

Reply:
I think we should switch everything over to paths eventually. For now I just wanted to be consistent with using strings.